priv/repo/demo.exs

```diff
@@ -11,3 +11,4 @@
 # and so on) as they will fail if something goes wrong.
 
 Lightning.Demo.setup(create_super: true)
+Lightning.Demo.tear_down(destroy_super: true)
```

Comment:
@taylordowns2000 @elias-ba I ran this script the other day and it leaves you with an empty db, are these two lines not in the wrong order?

Reply:
yeah totes. just fixed. thank you